docs/how-to/how-to-run-dind-with-kata.md

```diff
@@ -0,0 +1,122 @@
+# How to run Docker in Docker with Kata Containers
+
+This document describes the why and how behind running Docker-in-Docker ("dind") in a kata container.
+
+## Requirements
+
+- A working Kata Containers installation
+
+## Install and configure Kata Containers
+
+Follow the [Kata Containers installation guide](../docs/install/README.md) to Install Kata Containers on your Kubernetes cluster.
```

Comment:
```suggestion
Follow the [Kata Containers installation guide](/docs/install/README.md) to Install Kata Containers on your Kubernetes cluster.
```

Otherwise we link to `docs/docs/install/...`

Reply:
Or the simplest which is `../install/README.md`. The CI checks all markdown links so it will let you know if you get it wrong :smile:

Reply:
Ahh thanks for this -- will put in `../install/README.md`

- [ ] Fix reference to installation guide @ `how-to-run-dind-with-kata.md#L11`